New mission chain in Hai space (#4246)
endless-sky/endless-sky: data/hai missions.txt

	on complete
		payment 100000
		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`

mission "Nanachi 1"
	name "Escort Nanachi to <planet>"

> 	description "Help Nanachi, a Hai who just bought her first ship, make a simple delivery to Cloudfire."

Reviewer: ```suggestion
	description "Help Nanachi, a Hai who just bought her first ship, make a simple delivery to <destination>."
```

	minor
	source "Giverstone"
	destination "Cloudfire"
	deadline
	to offer
		"net worth" > 20000000
		"combat rating" > 1200
		year > 3018
	on offer
		conversation

> 			`	As you are making your way through the spaceport you notice a worried-looking Hai staring at the local job board and back at some sort of document they are holding.`

Reviewer: The first line in a conversation doesn't get tabbed in.
```suggestion
			`As you are making your way through the spaceport you notice a worried-looking Hai staring at the local job board and back at some sort of document they are holding.`
```
This needs changed throughout.

			choice
				`	(Approach the Hai.)`
				`	(Leave them alone.)`
					decline

> 			`	The Hai is a bit startled, but quickly calms down. "Sorry, you caught me by surprise. I've just bought my first ship and need to find some jobs to do, but nobody I've talked to wants to trust a beginner captain. I just don't know what to do anymore, I've been working for so long to qualify for this loan, but I'm running out of time and I'll never be able to pay it like this."`

Reviewer: Feels.

> 			`	Turns out the documents she is holding are the bank papers for the loan, somewhat similar to the one you took when you bought your first ship. You've come a long way since your days as a beginner, and know how hard it is to be starting out as a starship captain, you could probably convince someone here to offer a simple cargo job to this young Hai.`

Reviewer: ```suggestion
			`	Turns out the documents she is holding are the bank papers for the loan, somewhat similar to the one you took when you bought your first ship. You've come a long way since your days as a beginner, and know how hard it is to be starting out as a starship captain. You could probably convince someone here to offer a simple cargo job to this young Hai.`
```

			choice
				`	(Offer to help her find some jobs.)`
				`	(Wish her luck and leave.)`

> 					decline

Reviewer: I feel like this choice shouldn't just decline right away. Perhaps have a short line about Nanachi looking downtrodden that you didn't help as you walk away.
Glancing through the rest of the missions right now, you're also using a LOT of action choices. These choices here though could definitely be better off as dialog choices.

			`	Her ears twitch a bit, and her previously crestfallen eyes light up as she nearly jumps in joy at your offer. "Thank you so much captain! My name is Nanachi, I promise I won't cause any trouble at all, just a few quick jobs and I'm sure I'll be fine by myself!"`

> 			`	"Captain <first> <last>. Can you show me your ship so I can look for jobs it's better suited for?"`

Reviewer: You should indicate that this is the player speaking. This also gives me the same feel as a line in Zitchas' latest PR. Ref: https://github.com/endless-sky/endless-sky/pull/4222#discussion_r270873698. i.e. this seems like something that shouldn't be the player's dialog, instead something that is stated as having been said by the player, if that makes sense.
Perhaps change this line to something more like the following.
```suggestion
			`	You introduce yourself to Nanchi and shake her paw. "Can you show me your ship so I can look for jobs it's best suited for?" you ask.`
```

> 			`	She hesitates a bit, but leads you to her ship: you were expecting a small Hai ship, but to your surprise she shows you a rather old-looking Heavy Shuttle docked near your own ship.`

Reviewer: I believe this colon should be a semicolon, or just a full stop.

			`	"You took a loan to buy a human ship?" you ask.`

> 			`	She looks at the ground, as if ashamed of it. "It'd take way too long for me to qualify for a loan for even an Aphid, but I recently met a human merchant who wanted to sell his old ship; it was difficult but I managed to convince him to sell it to me instead of going to the shipyard office, I paid him its full price, but even for this one I had to take out the biggest loan I could. Its cargo hold isn't very impressive but it's got more bunks than our smallest ships, if that helps."`

Reviewer: ```suggestion
			`	She looks at the ground, as if ashamed of it. "It'd take way too long for me to qualify for a loan for even an Aphid, but I recently met a human merchant who wanted to sell his old ship; it was difficult, but I managed to convince him to sell it to me instead of going to the shipyard office, I paid him its full price, but even for this one I had to take out the biggest loan I could. Its cargo hold isn't very impressive but it's got more bunks than our smallest ships, if that helps."`
```

			choice
				`	(Help her check the job board.)`
				`	(Try approaching some people here looking for transport or for a courier.)`

> 			`	Turns out this isn't as easy as you thought it'd be, the spaceport is very empty, save for the occasional worker or drunken crewmember, and the desk workers by the job board refuse to even let you see the job list because Nanachi is with you, it seems her inability to get a job has given her a bad reputation here. Eventually, as you two begin scouting the edge of the docking area, you spot a troubled merchant near several crates. You approach him and he explains that his ship had some malfunctions and he cannot deliver this advanced surgical equipment to Cloudfire.`

Reviewer: ```suggestion
			`	Turns out this isn't as easy as you thought it'd be. The spaceport is very empty, save for the occasional worker or drunken crewmember, and the desk workers by the job board refuse to even let you see the job list because Nanachi is with you; it seems her inability to get a job has given her a bad reputation here. Eventually, as you two begin scouting the edge of the docking area, you spot a troubled merchant near several crates. You approach him and he explains that his ship had some malfunctions and he cannot deliver this advanced surgical equipment to <destination>.`
```

I'd change this up a bit, as it makes zero sense to me why Nanachi would be refused access to the jobs board. How about instead say that there are no available jobs small enough for Nanachi's shuttle, save for a single, dangerous job to deliver some goods to human space that Nanachi isn't interested in.

> 			`	"This is the worst! One jump away from the planet and the navigation system suddenly decides to go on strike! There's no way I can make it before the deadline!"`

Reviewer: Maybe have this line be part of the previous paragraph? e.g.
`You approach him and he explains that his ship has malfunctioned and he cannot deliver this advanced surgical equipment to Cloudfire. "This is the worst!" he yells in frustration. One jump away from the planet and the navigation system suddenly decides to go on strike! There's no way I can make it before the deadline!"`

Reviewer: Also, is this a human or a Hai merchant? Given that it could be either, I think it might be worth specifying.

> 			`	"My friend Nanachi here could deliver it for you. We were heading there anyway, a few extra crates won't make a difference." you say, trying to sound as convincing as possible.`

Reviewer: ```suggestion
			`	"My friend Nanachi here could deliver it for you. We were heading there anyway, a few extra crates won't make a difference," you say, trying to sound as convincing as possible.`
```

> 			`	"Look kid, Unfettered attacks make it so they need a lot of medical supplies in Cloudfire, and this here is the best medical equipment Hai space has to offer, these few crates could save many military lives and I don't think I should be trusting something like this to a beginner." he says.`

Reviewer: How does he know Nanachi is a beginner?
 
This response seems a little overly hostile when he was just given the chance to get his goods where they need to go on time. He basically has three choices with four outcomes: he repairs his ship first and gets the goods there too late, he gives up and the goods never get there, he gives them to Nanachi and she gets blown up, or he gives them to Nanachi and they make it to where they need to go. The only outcome that has the supplies getting to Cloudfire on time is giving them to Nanachi.

Maybe change this dialog to have this merchant pondering those options, ultimately deciding to risk the supplies with Nanachi as that at least means they'll get there on time. And remember, the player is escorting Nanachi, so it isn't like you're telling this guy that a single heavy shuttle (which hasn't even been mentioned) is flying on its own into dangerous territory to deliver some goods.

If you take that change, most of the rest of this conversation would be thrown out, so I'm just going to skip over it.

				accept
			label lie
			`	You go on about how Nanachi was the one who taught you how to fly a ship, and how famous she is amongst the Hai pilots, but as he looks at you with great disdain, he interrupts your stories. "Oh of course, an experienced captain that has been running around the spaceport for hours and can't even figure out the job board! I felt like I could maybe trust you to take care of her and the cargo, buy if you won't at least be honest with a fellow captain I can't put you in charge of a mission like this. Now get out of here, I have better things to do than waste my time listening to nonsense!" You and Nanachi continue to look for someone willing to give her a job, but after hours it's clear your best bet was with that man. Unable to find any other way to help her, you apologize and wish her the best of luck.`
				decline
	npc accompany save
		personality escort timid
		government "Hai"
		ship "Heavy Shuttle" "Acorn Chaser"
	on visit

> 		dialog `	You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`

Reviewer: `dialog` follows the same rules as `conversation`: the first one isn't tabbed in.
```suggestion
		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`
```

	on complete
		conversation

> 			`	Shortly after her ship lands, you meet up with Nanachi, who is doing her best to hide her excitement of being on another planet for the first time, gawking at some Hai military ships in particular. "So, how did you like your first time flying a ship?" you ask, to try and snap her out of it. "Oh um, I had a bit of trouble with the controls at first since they're a bit different from the ones in Hai ships, but I figured it out. I've always dreamt of being on other worlds, exploring and learning everything about them, I'm glad I'm finally able to do it." You help her get the crates out of the cargo hold and contact the client. Her share of the payment is small, as the client transfers most of the money to that merchant, but that doesn't make Nanachi any less excited about getting her first paycheck.`

Reviewer: `for the first time`
So you had the shuttle, but never left Giverstone until getting a job? How many days of looking for a job had passed for Nanachi to gain a "bad reputation". I already said how I'm not too keen on the whole idea of Nanachi having a bad rep just for looking for a job, so perhaps clarify in the on offer conversation that Nanachi only recently, say either today or the day before, bought the ship.

Should be a new line after `snap her out of it.`, since it's a new person's dialog.

`a bit different from the ones in Hai ships`
So this is Nanachi's first ship, but she knows what Hai ship controls are like? Might want to elaborate on that either here or in the on offer conversation when she mentions buying the shuttle.

I'd put a new line after `I'm glad I'm finally able to do it."`

`that merchant`
I feel like you should say "the merchant she got the job from" or something similar, instead of just `that merchant`.

```suggestion
			`	Shortly after her ship lands, you meet up with Nanachi, who is doing her best to hide her excitement of being on another planet for the first time, gawking at some Hai military ships in particular. "So, how did you like your first time flying a ship?" you ask, to try and snap her out of it. "Oh um, I had a bit of trouble with the controls at first since they're a bit different from the ones in Hai ships, but I figured it out. I've always dreamt of being on other worlds, exploring and learning everything about them. I'm glad I'm finally able to do it." You help her get the crates out of the cargo hold and contact the client. Her share of the payment is small, as the client transfers most of the money to that merchant, but that doesn't make Nanachi any less excited about getting her first paycheck.`
```

> 			`	"So, want to go to the spaceport and try to get a job on your own? I'll help you again in case you can't do it yet." you say. "Just give me an hour or two, back home our shipyard is kind of small, and I've never really gotten a chance to see a Shield Beetle up close, and there are many undergoing repairs here, I'll just go get a better look at them, I swear I'll be quick." You two agree to meet in the spaceport in a few hours, maybe this time she will do fine on her own.`

Reviewer: Again with needing a new line because of the person speaking switching. I'd also add a new line after Nanachi's dialog ends. i.e.
`"So..." you say.`
`"Just... quick"`
`You... own.`

```suggestion
			`	"So, want to go to the spaceport and try to get a job on your own? I'll help you again in case you can't do it yet." you say. "Just give me an hour or two. Back home our shipyard is kind of small. I've never really gotten a chance to see a Shield Beetle up close, and there are many undergoing repairs here. I'll just go get a better look at them, I swear I'll be quick." You two agree to meet in the spaceport in a few hours, maybe this time she will do fine on her own.`
```

mission "Nanachi 4"
	name "Protect Nanachi"
	description "Bring Nanachi's ship to Prime, keeping her safe from a pirate fleet that attacked a convoy heading to Hai space, then head back to <destination>."
	source "Allhome"
	stopover "Prime"
	deadline 14
	to offer
		has "Nanachi 3: done"
	on offer
		conversation
			`The bar is just as busy as the docks, many captains here must have transported something for the festival. "Captain <last>, captain Nanachi!" Narancia waves at you two and becons you to their table. You learn more about the festival, as they tell you that they were inspired by the one the Hai have during March, and wanted to bring small parts of human culture to Hai space.`
			`	"They can't go to our planets and experience it for themselves yet, so we thought about bringing it to them!" Vanessa says.`

> 			`	"It was very hard to get enough sponsors for this, but after years of hard work we even managed to have famous art pieces be brought here, some of which date back to the european renaissance!" Xavier adds.`

Reviewer 2: `european` needs to be capitalized

> 			`	Afterwards the conversation topic shifts to you as a captain, as you tell them about your journey after buying your first ship, and how you met Nanachi. You're about to order another drink for the group when a grizzled old man barges into the bar, scouting the area until he spots your group, and starts shouting: "The art convoy has been attacked! The freighter managed to escape on time but they're stranded on Prime now, the pirates are patrolling the route to the wormhole!"`

Reviewer 2: `tell them about` -> `describe`

`scouting the area until he spots your group` seems unnecessary

`escape on time` -> `escape`

`Prime now, the pirates are patrolling the route to the wormhole!` -> `Prime now, and the pirates are still out there, patrolling the route to the wormhole!`

> 			`	Xavier stands up, with a shocked expression. "How? We had a Leviathan escorting them!"`

Reviewer 2: `up, with` -> `up with`

> 			`	After catching his breath, the man at the door responds. "The pirates must have been stealing Hai tech from ships going back home, the freighter's captain said they shredded the escort!" Chaos spreads amongst the other captains, and the group sitting with you is in anguish.`

Reviewer 2: `home, the` -> `home. The`

			choice
				`	"Are those art pieces really that valuable?"`
				`	"Can't you just postpone the festival until the pirates are taken care of?"`

> 			`	Xavier collapses back to the table, holding his head in both hands. "Those paintings and sculptures were our main attraction! They're worth dozens of millions! It'll take a month before the Navy ever notices the pirates there and make it safe for the convoy to come here. The art exhibition is scheduled two weeks from now!"`

Reviewer 2: `dozens of millions` -> tens of millions`

`make it safe` -> `makes it safe`

> 			`	As they try to figure out what to do, Nanachi speaks up. "I'll go there, I pilot a human ship and if we just need to transfer cargo from one ship to the other, I could get out of there quickly, nobody would notice that a Hai ever went there."`

Reviewer 2: `go there, I pilot` -> `go there. I pilot`

`there quickly, nobody would` -> `there quickly. Nobody would`

> 			`	"Are you insane? If they managed to take down that Leviathan, you wouldn't last a second! Besides, our convoy was extremely fast, if even they were caught your chances of outrunning the pirates are next to nothing!" Vanessa says.`

Reviewer 2: I would move the `Vanessa says` to right after `"Are you insane?"`

`extremely fast, if even` -> `extremely fast. If even`

> 			`"Captain <last>," Nanachi turns to you. "I know you already helped me way more than anyone would ever be willing to, and I'm really grateful for that, but would you be willing to help me one last time, to get to that planet, and travel back here?" While it's certain that for a pirate fleet to take down a Leviathan so easily they must be incredibly powerful, and this could be extremely dangerous, you're sure that if Nanachi tries to get to Prime by herself she'll just get herself killed.`

Reviewer 2: `way more` is something children say; maybe replace this with something else. Also this is a huge run-on sentence and should be split up.

			choice
				`	"Alright, I'll make sure we reach Prime and get back here safely."`

> 				`	"No, I agreed to help you find and deal with some simple jobs, I can't risk fighting such a fleet by myself."`

Reviewer 2: `simple jobs, I can't` -> `simple jobs, but I can't`

					decline

> 			`	She thanks you profusely, and promises that she'll pay you for everything as soon as she can. The organizers are a bit hesitant, but seeing as no other captain here decided to offer them help, they agree to let you two go to Prime retrieve the cargo.`

Reviewer 2: `here decided` -> `here has decided`

`you two go to Prime retrieve the cargo` -> `the two of you go to Prime and retrieve the cargo`

> 			`	As you're leaving the bar, Xavier pulls you aside, speaking very quietly, "<last>, I know you're just trying to help this girl, but don't get killed over this, if you run into those pirates don't try to fight them, just rush to Prime."`

Reviewer 2: `over this, if you` -> `over this. If you`

				accept
	npc
		personality nemesis waiting
		government "Pirate"
		system "Betelgeuse"
		ship "Marauder Firebird (Hai)" "Nut King Call"
		ship "Marauder Splinter (Hai)" "The Nutcracker"
		ship "Marauder Quicksilver (Hai)" "Hazelnut's Bane"
		ship "Marauder Bounder (Hai)" "Rodent Hunter"

> 	npc accompany save

Reviewer 2: I get that it's to make the mission more challenging, but it makes no sense at all that Nanachi would follow you in her ship when it has no combat power and would only make the situation vastly more dangerous for her.

		personality escort timid
		government "Hai"
		ship "Heavy Shuttle" "Acorn Chaser"
	on visit

> 		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`

Reviewer 2: `them` -> `her`

	on stopover "Prime"
		conversation
			`	The convoy's captain messaged you as you entered the system and told you to land on a remote, hidden area of Prime, where they have already prepared for all the cargo to be transfered. They refuel your ships and urge you two to go back. "Captain Nanachi can stay hidden with us here if you need supplies or anything from the outfitter, but please don't take too long."`
	on complete
		event "Nanachi: payment" 180
		conversation

> 			`Landing back on Allhome, the festival group gives you and Nanachi a warm hug, "We sold tens of thousands of tickets for the festival by promising to host a great art exhibition with pieces from all over human space, without them it'd be a complete failure, and we'd be in serious trouble with our sponsors." Narancia says.`

Reviewer 2: `human space, without them` -> `human space. Without them`

> 			`	As the cargo is rushed to the exhibition site, Nanachi shows you the chip she got as payment, and the balance of her bank account: this last payment allowed her to fully pay off her loan. "Captain," she speaks with some difficulty, and tries to hide her face, as if to wipe away some tears. "I'll never be able to thank you properly, if not for you I'd sill be lost in that spaceport, and would eventually have to go back to my mining job. I'll work hard now, I'll save up and pay you for everything!" She hugs you and bows to you in a way you'd see in movies, as if trying to immitate the gesture. You two finally say goodbye, and head back to your ships.`

Reviewer 2: `sill` -> `still`

event "Nanachi: payment"

mission "Nanachi: Payment"
	landing
	source
		government "Hai"
		not planet "Darkwaste"
	to offer
		has "event: Nanachi: payment"
	on offer
		payment 700000
		event "Nanachi: Update" 720
		conversation

> 			`Shortly after you land, you get a message from Nanachi, the young hai pilot who you helped start out. "Dear captain <last>, I hope you're doing well! As I promised, ever since we parted ways I've been working towards getting enough money to pay you for being so kind to help me. I've sent you <payment>. I'm deeply grateful to you for helping make my dream come true."`

Reviewer 2: `helped start out` -> `helped to become a captain`, `showed the ropes`, etc.

> 			`Shortly after you land, you get a message from Nanachi, the young hai pilot who you helped start out. "Dear captain <last>, I hope you're doing well! As I promised, ever since we parted ways I've been working towards getting enough money to pay you for being so kind to help me. I've sent you <payment>. I'm deeply grateful to you for helping make my dream come true."`

Reviewer 3: `captain <last>` -> `Captain <last>`

				decline

event "Nanachi: Update"

mission "Nanachi Meeting"
	landing
	source
		government "Hai"
		not planet "Darkwaste"
	to offer
		has "event: Nanachi: Update"
		random > 30
	on offer
		conversation

> 			`After you leave your ship and start making your way through the docking area, you notice a Hai running towards you and waving. As they get closer, you realize that it's Nanachi, the Hai pilot who you helped out years ago. "I was just about to go back to my crew when I saw you coming out of your ship. I know a nice snack bar here, let's go there!" Before you can even say hello she's dragged you to the bar, and ordered some drinks for the two of you. She tells you how the last years have been, and of her experiences as a captain. She has long since sold her old ship, and now pilots a Geocoris, transporting large amounts of cargo across Hai space. You also tell her about how your life has been ever since you last met, of your adventures and how far you've come. You keep swapping stories for hours, until the bar is ready to close.`

Reviewer 2: `She tells you how the last years have been, and of her experiences as a captain` -> `She tells you about her last few years as a captain`

> 			`	You thank her for the drinks, and get ready to say your goodbyes. "Oh wait, before you go captain, let me give you something." She says, right as you're about to head to your ship, and grabs a small pendant from her pocket. "Back home in Giverstone, when I had some free time from the mining works, I took a hobby of sculpting some small stones into ship miniatures. I thought it'd be nice to make this for you in case I saw you again." She gives you the pendant, which has been sculpted into a Hai Shield Beetle, it is extremely well made. "It's just a good-luck charm I thought you'd like. Thanks again captain!" You two part ways, and head back to your ships.`

Reviewer 2: `something." She` -> `something," She`

`your ship, and grabs` -> `your ship. She grabs`

`I thought it'd be nice to make this for you in case I saw you again."` But wasn't she a miner before she met you? How could she have sculpted this for you while she was still a miner?

			choice
				`	"Hey, what is going on?"`
			`	You introduce yourself and the parents tell you that they've been on vacation here and need transport back to Hai-home. "Don't take this the wrong way, but captain Nanachi told us her ship is one of your human ships, a Heavy Shuttle was it? It has enough bunks for us but our sons already caused a ruckus on the way here in a Hai ship, we're worried that since they aren't used to the accomodations they might end up giving her too much trouble."`
			`	You reassure them that the trip will be very quick, and that the ship's interior is somewhat similar to that of Hai ships, until they finally agree on going with her. They settle on the pay and you two help the Hai get their baggages to Nanachi's ship. When the family is focused on a conversation, you quietly ask Nanachi what was the issue she was having with the other people she encountered.`

> 			`	"They all seemed willing to give me a job, but I ended up blowing it when I asked for more money, I just really need to get out of debt so I guess I went a bit overboard." she tries to play it cool, but it's clear that she is embarrased of losing so many potential jobs. When you reach her ship you take a look at the prices for commodities in Hai-home, and tell Nanachi to stock up on heavy metals to sell there, to help her make a bit more profit.`

Reviewer 4: "baggage" not "baggages", we're not hobbitses.
Capitalize "she" in the last paragraph, after `overboard."`
The last sentence in the last paragraph is too long, try to break it up.
